### DISCUSSION TITLE
display birdnet-pi on local hard raspberry screen

### DISCUSSION THREAD

**Discussion by sangukoo** [mcguirepr89/BirdNET-Pi#1221] [Q&A]
Bonjour,
J'avais une station birdnet-pi sur raspberry pi 4 qui fonctionnait vraiment bien avec un affichage local sur l'écran 7 " du raspberry directement au démarrage.
Ma carte SD a craché et j'ai du tout réinstaller.
tout fonctionne bien en distant depuis mon pc je peux voir le site http://birdnetpi1.local/ et les détection se passe bien.
J'aimerai retrouver le fonctionnement passé avec l'affichage en local sur l'écran 7 " et malgrés de nombreuse recherche je n'ai pas de solution.
Quelqu’un peut il m'aider
Merci

Good morning,
I had a birdnet-pi station on raspberry pi 4 which worked really well with a local display on the 7 "screen of the raspberry directly on startup.
My SD card crashed and I had to reinstall everything.
everything works well remotely from my PC I can see the site http://birdnetpi1.local/ and the detection is going well.
I would like to find the past operation with the local display on the 7 "screen and despite numerous searches I have no solution.
Can anyone help me
THANKS

**Comment by Nachtzuster** [comment]
I guess I would try the following:
1. Full version od Raspberry OS (ie with desktop)
2. auto-login your user
3. auto-start a browser with `firefox --kiosk http;//localhost`. Not quite sure how RaspOS' desktop does autostart applications though

**Reply by alexbelgium** [reply]
To autostart your kiosk mode (considering you have installed firefox) you could write in a script named (for example) autostart.sh the line proposed by Nachtzuster : 
```
#!/bin/bash
firefox --kiosk http://localhost
```
add execute permission with `chmod +x autostart.sh` then type `crontab -e` and type at bottom `@reboot /home/$USER/autostart.sh` (you need to change $USER with your own user)
